applications/FluidDynamicsApplication/custom_utilities/fluid_element_data.h

```diff
@@ -43,6 +43,8 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) FluidElementData
 
     using NodalVectorData = BoundedMatrix<double,TNumNodes,TDim>;
 
+    using NodalTensorData = std::vector< Matrix >;
```

Comment:
Not sure if we should expose the `Matrix` usage. For the sake of performance, I'd stick to `BoundedMatrix` until it is explicitly required. Note that this wouldn't be an issue if was available partial template specialization #6611.